app/jobs/cangaroo/job.rb

```diff
@@ -31,6 +32,10 @@ def restart_flow(response)
       # if no json was returned, the response should be discarded
       return if response.blank?
 
+      if !self.process_response
```

Comment:
Why do you not use `return unless self.process_response`? 
Personally I prefer to follow the [ruby-style-guide](https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)